Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md

```diff
@@ -2175,4 +2178,235 @@ There is no context output for this command.
 You will be automatically redirected to a link with the following structure:
 >```REDIRECT_URI?code=AUTH_CODE&session_state=SESSION_STATE```
 >2. Copy the `AUTH_CODE` (without the `code=` prefix, and the `session_state` parameter)
-and paste it in your instance configuration under the **Authorization code** parameter.
\ No newline at end of file
+and paste it in your instance configuration under the **Authorization code** parameter.
+
+### msg-advanced-hunting
+
+***
+Advanced hunting is a threat-hunting tool that uses specially constructed queries to examine the past 30 days of event data in Microsoft Graph Security.
```

Comment:
add information about the option to save the result to both contexts.

Reply:
and add the new param (context) to the configuration section with explanation the use of the param

Reply:
I added